web_development_101/git_intro.md

```diff
@@ -59,7 +59,7 @@ But for now, in this lesson, try to understand what Git is and why it is so powe
   1. Read chapter 1 in this book about [version control](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control) and appreciate the differences between local, centralised and distributed version control systems.
   2. Now watch this [video](https://www.youtube.com/watch?v=8oRjP8yj2Wo) about how git can improve the workflow of an individual or a team of developers.
   3. Get some [context](https://www.youtube.com/watch?v=1h9_cB9mPT8&feature=youtu.be&t=13s) of how git and GitHub came about. Make sure you know the difference between the two: note how Git is referred to as a technology used in the *command line* while GitHub is a *website* you can [visit](https://github.com/)
-  4. If you haven't, install Git on your computer. Remember this is one of the steps of [RailsBridge Installfest](http://installfest.railsbridge.org/installfest/).
+  4. If you haven't, install Git on your computer. Remember this is one of the steps of our [Installation Prerequisits](https://www.theodinproject.com/courses/web-development-101/lessons/prerequisites).
```

Comment:
This links only to the Windows Subsystem installation of git. The git installation guide for UNIX systems is on this [link](https://www.theodinproject.com/courses/web-development-101/lessons/your-first-rails-application?ref=lnav)